### ISSUE TITLE
Uncaught (in promise) SyntaxError: The requested module '/@modules/element-plus.js' does not provide an export named 'ElMessage'

### ISSUE THREAD

**Issue by em2046** [element-plus/element-plus#667]
https://element-plus.org/#/en-US/component/message
Local import
import { ElMessage } from 'element-plus';

error: Uncaught (in promise) SyntaxError: The requested module '/@modules/element-plus.js' does not provide an export named 'ElMessage'

if using
import ElMessage from 'element-plus/lib/el-message';
error: Uncaught (in promise) SyntaxError: The requested module '/@modules/element-plus/lib/utils/util.js' does not provide an export named 'isVNode'

    "element-plus": "^1.0.1-alpha.20",
    "vue": "^3.0.2",
    "vite": "^1.0.0-rc.9"

**Comment by zazzaz** [comment]
This might be Vite bug, ref here https://github.com/vitejs/vite/pull/837.

You can use Webpack bundle at this moment before Vite fix it.

**Comment by lqsong** [comment]
> This might be Vite bug, ref here [vitejs/vite#837](https://github.com/vitejs/vite/pull/837).
> 
> You can use Webpack bundle at this moment before Vite fix it.

```ts
import { ElMessage } from 'element-plus';
```
tip:
Module '"../../../../node_modules/element-plus/types"' has no exported member 'ElMessage'. Did you mean 'Message'?Vetur(2724)

**Comment by zazzaz** [comment]
fixed in 1.0.1-alpha.23

you can use
```
import { ElMessage } from 'element-plus';
```

**Comment by zazzaz** [comment]
still not fixed

**Comment by JeremyWuuuuu** [comment]
@zazzaz  Is this fixed in #577 ?

**Comment by zazzaz** [comment]
fixed in  #577
